src/hotspot/share/cds/archiveUtils.cpp

```diff
@@ -271,16 +272,16 @@ void DumpRegion::append_intptr_t(intptr_t n, bool need_to_mark) {
 
 void DumpRegion::print(size_t total_bytes) const {
   char* base = used() > 0 ? ArchiveBuilder::current()->to_requested(_base) : nullptr;
-  log_debug(cds)("%s space: %9zu [ %4.1f%% of total] out of %9zu bytes [%5.1f%% used] at " INTPTR_FORMAT,
+  log_debug(aot)("%s space: %9zu [ %4.1f%% of total] out of %9zu bytes [%5.1f%% used] at " INTPTR_FORMAT,
                  _name, used(), percent_of(used(), total_bytes), reserved(), percent_of(used(), reserved()),
                  p2i(base));
 }
 
 void DumpRegion::print_out_of_space_msg(const char* failing_region, size_t needed_bytes) {
-  log_error(cds)("[%-8s] " PTR_FORMAT " - " PTR_FORMAT " capacity =%9d, allocated =%9d",
+  aot_log_error(aot)("[%-8s] " PTR_FORMAT " - " PTR_FORMAT " capacity =%9d, allocated =%9d",
                  _name, p2i(_base), p2i(_top), int(_end - _base), int(_top - _base));
```

Comment:
Please fix the indentation of line 282.

Reply:
Many lines are now out of alignment. I didn't change any of such lines to avoid introducing massive diffs. Also, in a short time (maybe in 6 months) we will change these from `aot_log_xxx(aot)` back to  `log_xxx(aot)` :

```
log_error(aot)("[%-8s] " PTR_FORMAT " - " PTR_FORMAT " capacity =%9d, allocated =%9d",
```

and I don't want to redo all the line alignments again.